content/en/docs/started/installing-kubeflow.md

```diff
@@ -9,140 +9,190 @@ Kubeflow is an end-to-end Machine Learning (ML) platform for Kubernetes, it prov
-        <td><a href="/docs/distributions/openshift/">Docs</a></td>
-        <td><a href="https://opendatahub.io/docs/kubeflow.html">External Website</a></td>
-      </tr>
-      <tr>
-        <td>Argoflow</td>
-        <td>Argoflow Community</td>
-        <td>Conformant Kubernetes</td>
-        <td>1.3</td>
-        <td>N/A</td>
-        <td><a href="https://github.com/argoflow/argoflow">External Website</a></td>
+        <td>
+          ➤ OpenShift
+        </td>
+        <td>
+          ➤ <a href="https://opendatahub.io/docs/kubeflow.html">Official Website</a>
+          <br>
+          ➤ <a href="/docs/distributions/openshift/">Other Docs</a>
+        </td>
       </tr>
       <tr>
         <td>Arrikto Kubeflow as a Service</td>
-        <td>Arrikto</td>
-        <td>Fully Managed</td>
-        <td>1.4</td>
-        <td>N/A</td>
-        <td><a href="http://kubeflow.arrikto.com/">External Website</a></td>
+        <td rowspan="2">Arrikto</td>
+        <td>
+          ➤ N/A (Fully Managed)
+        </td>
+        <td>
+          ➤ <a href="https://www.arrikto.com/kubeflow-as-a-service/">Official Website</a>
+        </td>
       </tr>
       <tr>
         <td>Arrikto Enterprise Kubeflow</td>
-        <td>Arrikto</td>
-        <td>EKS,
-            AKS,
-            GKE
-        </td>
-        <td>1.4</td>
         <td>
-          <a href="/docs/distributions/ekf/">Docs</a>
+          ➤ Amazon Elastic Kubernetes Service (EKS)
+          <br>
+          ➤ Azure Kubernetes Service (AKS)
+          <br>
+          ➤ Google Kubernetes Engine (GKE)
         </td>
         <td>
-          <a href="https://www.arrikto.com/enterprise-kubeflow/">External Website</a>
+          ➤ <a href="https://www.arrikto.com/enterprise-kubeflow/">Official Website</a>
         </td>
       </tr>
       <tr>
         <td>Charmed Kubeflow</td>
         <td>Canonical</td>
-        <td>Conformant Kubernetes</td>
-        <td>1.6</td>
-        <td><a href="/docs/distributions/charmed/">Docs</a></td>
-        <td><a href="https://charmed-kubeflow.io/">External Website</a></td>
+        <td>
+          ➤ All Conformant Kubernetes
+        </td>
+        <td>
+          ➤ <a href="https://charmed-kubeflow.io/">Official Website</a>
+        </td>
       </tr>
       <tr>
-        <td>Kubeflow on Oracle Container Engine for Kubernetes</td>
+        <td>Kubeflow on Oracle Cloud Infrastructure</td>
         <td>Oracle</td>
-        <td>Oracle Cloud Infrastructure (OCI)</td>
-        <td>1.6</td>
-        <td>N/A</td>
-        <td><a href="https://github.com/oracle-devrel/kubeflow-oke">External Website</a></td>
+        <td>
+          ➤ Oracle Container Engine for Kubernetes (OKE)</td>
+        <td>
+          ➤ <a href="https://github.com/oracle-devrel/kubeflow-oke">Official Website</a>
+        </td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+### Legacy Distributions
+
+The following table lists some <b>distributions which provide versions older than `Kuebflow 1.6`</b>.
```

Comment:
```suggestion
The following table lists some <b>distributions which provide versions older than `Kubeflow 1.6`</b>.
```

Reply:
Thanks, I have resolved this typo in https://github.com/kubeflow/website/pull/3381/commits/79e0d92d3b5e9054a2ed84e84b8fdf3fda87b08c.